common.gypi

```diff
@@ -469,31 +454,22 @@
             'ldflags!': [ '-pthread' ],
           }],
           [ 'OS=="aix"', {
-            'variables': {'real_os_name': '<!(uname -s)',},
+            'cflags': [ '-maix64' ],
+            'ldflags!': [ '-rdynamic' ],
+            'ldflags': [ '-Wl,-bbigtoc' ],
             'conditions': [
-              [ 'target_arch=="ppc"', {
-                'ldflags': [
-                  '-Wl,-bmaxdata:0x60000000/dsa',
-                  '-Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/pthread',
-                 ],
-              }],
-              [ 'target_arch=="ppc64"', {
-                'cflags': [ '-maix64' ],
-                'ldflags': [
-                  '-maix64',
-                  '-Wl,-blibpath:/usr/lib:/lib:'
-                    '/opt/freeware/lib/pthread/ppc64',
-                ],
-              }],
-              ['"<(real_os_name)"=="OS400"', {
+              [ '">(aix_variant_name)"=="OS400"', {
```

Comment:
The `>` tells GYP to use ["late variable expansion"](http://gyp3.org/docs/InputFormatReference.html#variable-expansions----), which is required because this is in a `target_defaults` clause which is resolved before the top level `variables` clause. Using a `<` for early resolution fails GYP ¯\\_(ツ)_/¯

If after refactoring everything we can reduce the usage of `aix_variant_name`, I'll try to move it back to a closer scope.

Reply:
Done